assets/src/infrastructure/ui/display/steps/Steps.tsx

```diff
@@ -14,26 +15,31 @@ const Steps: React.FC<StepsProps> = ({
 	...props
 }) => {
 	const wrapperClassName = classNames(props.className, 'ee-form-steps__wrapper');
+
 	const listClassName = classNames('ee-form-steps', {
 		'ee-form-steps--vertical': vertical,
 		'ee-form-steps--inline': !vertical,
 	});
+
 	const heading = props.heading ?? __('Steps');
+
 	return (
-		<div className={ wrapperClassName }>
-			<h3>{ heading }</h3>
-			<ul className={ listClassName }>
-				{ Children.map(children, (child: any, index) => {
+		<div className={wrapperClassName}>
+			<Heading as='h3' mb={{ base: 4, md: 0 }}>
```

Comment:
this was too close on mobile:
![image](https://user-images.githubusercontent.com/1477453/82566477-a38aea80-9b84-11ea-9053-dd08fcb447e0.png)

Reply:
the fix: 

![image](https://user-images.githubusercontent.com/1477453/82566517-b1407000-9b84-11ea-976c-bf0f5815f4bf.png)